middleware.go

```diff
@@ -231,11 +231,11 @@ func requestLogger(c *Client, r *Request) error {
 		}
 
 		reqLog := "\n---------------------- REQUEST LOG -----------------------\n" +
-			fmt.Sprintf("%s  %s  %s\n", r.Method, rr.URL.RequestURI(), rr.Proto) +
-			fmt.Sprintf("HOST   : %s\n", rr.URL.Host) +
-			fmt.Sprintf("HEADERS:\n") +
-			composeHeaders(rl.Header) + "\n" +
-			fmt.Sprintf("BODY   :\n%v\n", rl.Body) +
+			fmt.Sprintf("%s %s %s\n", r.Method, rr.URL.RequestURI(), rr.Proto) +
+			fmt.Sprintf("HOST: %s\n", rr.URL.Host) +
+			fmt.Sprintf("HEADERS:\n%s\n", composeHeaders(rl.Header)) +
+			fmt.Sprintf("COOKIES:\n%s\n", composeCookies(c.GetClient().Jar, *rr.URL)) +
```

Comment:
@matthewpoer Just remembered one thing. This method focuses on Cookies from Jar and creates a section called `COOKIES`. Since cookies get separate section in the debug log. I think its better to read request header cookies added by the user into the cookies section finally remove the request header `Cookie` from headers section. Let me know your thoughts.